lawlibrary/templates/peachjam/about.html

```diff
@@ -0,0 +1,41 @@
+{% extends "peachjam/layouts/main.html" %}
+{% load i18n %}
+
+{% block title %} {% trans 'About' %}{% endblock %}
+
+{% block page-content %}
+<section class="py-5">
+  <div class="container">
+    <h1 class="mb-4">{% trans 'About the platform' %}</h1>
+    <p>
+      {% blocktrans %}
```

Comment:
```suggestion
      {% blocktrans trimmed %}
```

Reply:
This strips the whitespace around the paragraphs and makes the translation clearer in crowdin.